### PR TITLE
[ZEPPELIN-6005] Update Kyuubi JDBC docs

### PR DESCRIPTION
**Pull request by pan3793**
### What is this PR for?

Update the outdated Kyuubi JDBC docs
https://kyuubi.readthedocs.io/en/master/client/jdbc/kyuubi_jdbc.html

### What type of PR is it?

Documentation

### Todos
* [ ] - Task

### What is the Jira issue?

ZEPPELIN-6005

### How should this be tested?

Review

### Questions:
* Does the license files need to update? No.
* Is there breaking changes for older versions? No.
* Does this needs documentation? Yes

### CODE REVIEW

**Comment by pan3793** [comment]
cc @huage1994

File: docs/interpreter/jdbc.md [hunk 1]
```diff
@@ -853,7 +853,7 @@ Properties
   </tr>
   <tr>
     <td>default.url</td>
-    <td>jdbc:hive2://kyuubi-server:10009</td>
+    <td>jdbc:kyuubi://kyuubi-server:10009</td>
```
Comment (pan3793) [inline docs/interpreter/jdbc.md:856]: both `jdbc:hive2//` and `jdbc:kyuubi//` work, but `jdbc:kyuubi//` is recommended

File: docs/interpreter/jdbc.md [hunk 2]
```diff
@@ -865,11 +865,7 @@ Dependencies
     <th>Excludes</th>
   </tr>
   <tr>
-    <td>org.apache.kyuubi:kyuubi-hive-jdbc-shaded:1.6.1-incubating</td>
-    <td></td>
-  </tr>
-  <tr>
-    <td>org.apache.hive:hive-jdbc:3.1.2</td>
+    <td>org.apache.kyuubi:kyuubi-hive-jdbc-shaded:1.9.0</td>
```
Comment (pan3793) [inline docs/interpreter/jdbc.md:868]: previous might be a mistake, the `kyuubi-hive-jdbc-shaded` is an all-in-one all, we don't need the `hive-jdbc`

File: docs/interpreter/jdbc.md [hunk 3]
```diff
@@ -916,11 +912,15 @@ Dependencies
     <th>Excludes</th>
   </tr>
   <tr>
-    <td>org.apache.kyuubi:kyuubi-hive-jdbc-shaded:1.6.1-incubating</td>
+    <td>org.apache.kyuubi:kyuubi-hive-jdbc-shaded:1.9.0</td>
     <td></td>
   </tr>
   <tr>
-    <td>org.apache.hive:hive-jdbc:3.1.2</td>
+    <td>org.apache.hadoop:hadoop-client-api:3.3.6</td>
```
Comment (pan3793) [inline docs/interpreter/jdbc.md:919]: There are several approaches(including Hadoop UGI) to use Kerberos with `kyuubi-hive-jdbc-shaded`, the current Zeppelin JDBC Interpreter only implements the UGI approach, so Hadoop deps are required. I can extend the Zeppelin code to support the JDK JAAS based Kerberos authentication to get rid of Hadoop deps in the future

**Review by huage1994 — APPROVED** [review]

**Pull request merged**